src/platform/packages/shared/kbn-esql-validation-autocomplete/src/autocomplete/__tests__/autocomplete.command.test.ts

```diff
@@ -0,0 +1,267 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
```

Comment:
Since we're now defining the tests here, I've recreated the stats cases according to the new testing rules and added new tests that highlight the differences with inlinestats